src/qml/main.qml

```diff
@@ -135,40 +190,49 @@ ApplicationWindow {
                 leftAction: backAction
             }
         }
-
         Component {
             id: _gameSetup
             GameSetup {
                 leftAction: drawerAction
             }
         }
-
         Component {
             id: _rulebook
             Rulebook {
-                leftAction: backAction
+                leftAction: drawerAction
             }
         }
-
         Component {
             id: _lore
             Lore {
-                leftAction: backAction
-            }
-        }
-
-        Component {
-            id: _language
-            Language {
-                leftAction: backAction
+                leftAction: drawerAction
             }
         }
+    }
 
-        Component {
-            id: _credits
-            Credits {
-                leftAction: backAction
+    StateGroup {
+        id: pages
```

Comment:
This `id` is not clear enough, when referenced around I thougth it was a list of `Page` components. Change it to something more meaningful.